extendeddatacrossword_test.go

```diff
@@ -223,104 +194,103 @@ func TestCorruptedEdsReturnsErrByzantineData(t *testing.T) {
 		},
 	}
 
-	for codecName, codec := range codecs {
-		t.Run(codecName, func(t *testing.T) {
-			for _, test := range tests {
-				t.Run(test.name, func(t *testing.T) {
-					eds := createTestEds(codec, shareSize)
+	codec := NewLeoRSCodec()
 
-					// compute the rowRoots prior to corruption
-					rowRoots, err := eds.getRowRoots()
-					assert.NoError(t, err)
+	t.Run(codec.Name(), func(t *testing.T) {
+		for _, test := range tests {
+			t.Run(test.name, func(t *testing.T) {
+				eds := createTestEds(codec, shareSize)
 
-					// compute the colRoots prior to corruption
-					colRoots, err := eds.getColRoots()
-					assert.NoError(t, err)
+				// compute the rowRoots prior to corruption
+				rowRoots, err := eds.getRowRoots()
+				assert.NoError(t, err)
 
-					for i, coords := range test.coords {
-						x := coords[0]
-						y := coords[1]
-						eds.setCell(x, y, test.values[i])
-					}
+				// compute the colRoots prior to corruption
+				colRoots, err := eds.getColRoots()
+				assert.NoError(t, err)
 
-					err = eds.Repair(rowRoots, colRoots)
-					assert.Error(t, err)
-
-					// due to parallelisation, the ErrByzantineData axis may be either row or col
-					var byzData *ErrByzantineData
-					assert.ErrorAs(t, err, &byzData, "did not return a ErrByzantineData for a bad col or row")
-					assert.NotEmpty(t, byzData.Shares)
-					assert.Contains(t, byzData.Shares, corruptChunk)
-				})
-			}
-		})
-	}
+				for i, coords := range test.coords {
+					x := coords[0]
+					y := coords[1]
+					eds.setCell(x, y, test.values[i])
+				}
+
+				err = eds.Repair(rowRoots, colRoots)
+				assert.Error(t, err)
+
+				// due to parallelisation, the ErrByzantineData axis may be either row or col
+				var byzData *ErrByzantineData
+				assert.ErrorAs(t, err, &byzData, "did not return a ErrByzantineData for a bad col or row")
+				assert.NotEmpty(t, byzData.Shares)
+				assert.Contains(t, byzData.Shares, corruptChunk)
+			})
+		}
+	})
```

Comment:
[optional] this could remove the outer `t.Run(codec.Name()` because this test only tests one codec